ui/src/components/sermons/SermonCard.js

```diff
@@ -35,13 +58,33 @@ const SermonCard = ({ sermonData, allSermons }) => {
             }}
           >
             <AspectRatio width="100%" ratio={16 / 9}>
-              <Image borderTopRadius="20" src={sermonImage} objectFit="cover" />
+              <>
+                <Image
+                  borderTopRadius="20"
+                  src={sermonImage}
+                  objectFit="cover"
+                />
+                {onlineSermon && (
+                  <Box pr={'65%'} pb={'40%'}>
+                    <HStack
+                      px={{ base: '0', md: '2', lg: '3' }}
+                      color="white"
+                      bg="red.500"
+                      boxShadow="dark-lg"
+                    >
+                      <Text fontWeight={'800'}>
+                        <CircleIcon /> LIVE
+                      </Text>
+                    </HStack>
+                  </Box>
+                )}
+              </>
             </AspectRatio>
             <Box position="absolute" left="100%" top="50%">
               <AspectRatio width="20%" ratio={1 / 1}>
                 <Image
                   borderRadius="100%"
-                  src={process.env.PUBLIC_URL + "/images/PlayButton.png"}
+                  src={process.env.PUBLIC_URL + '/images/PlayButton.png'}
```

Comment:
[NO-ACTION] lol this file man.. i just realised that idk what this does nor does it show up at all in website.. no need to action on this, just commenting haha

Reply:
lmao...